controllers/provider-openstack/pkg/controller/controlplane/valuesprovider.go

```diff
@@ -229,9 +231,13 @@ func getConfigChartValues(
 	values := map[string]interface{}{
 		"kubernetesVersion": cluster.Shoot.Spec.Kubernetes.Version,
 		"domainName":        c.DomainName,
+		"domainID":          c.DomainID,
 		"tenantName":        c.TenantName,
+		"tenantID":          c.TenantID,
 		"username":          c.Username,
 		"password":          c.Password,
+		"userDomainID":      c.UserDomainID,
```

Comment:
Why are these added here? According to below comparison they are only valid for a certain Kubernetes version and should only be added if the version constraint matches?

Reply:
No, the domain and tenant ID is available for old cloud provider versions as well.